images/router/haproxy/conf/haproxy-config.template

```diff
@@ -6,7 +6,8 @@
 {{ define "/var/lib/haproxy/conf/haproxy.config" }}
 {{ $workingDir := .WorkingDir }}
 global
-  # maxconn 4096
+  maxconn {{env "ROUTER_MAX_CONNECTIONS" "20000"}}
```

Comment:
At the time I made the comment, the theory about why the tests were breaking was that the AWS file handle limits were not allowing 200k.  We had dropped everything to 2k in oadm, so this was defaulting the existing routers to a higher number.

We found that the AWS limit was huge and not causing the problem (rather it was a default coming from 'oc cluster up' that was not set, so setting maxconns to 0).  So this comment is stale.